src/test/java/io/vlingo/wire/fdx/bidirectional/netty/client/NettyClientRequestResponseChannelTest.java

```diff
@@ -1,188 +1,189 @@
-                                        final NioEventLoopGroup childGroup, final int testPort) throws InterruptedException {
-    final ServerBootstrap b = new ServerBootstrap();
-
-    return b.group(parentGroup, childGroup)
-            .channel(NioServerSocketChannel.class)
-            .childHandler(new ChannelInitializer<SocketChannel>() {
-              @Override
-              public void initChannel(SocketChannel ch) throws Exception {
-                ch.pipeline()
-                  .addLast(new ChannelInboundHandlerAdapter() {
-                    private byte[] partialRequestBytes;
-
-                    @Override
-                    public void channelActive(final ChannelHandlerContext ctx) throws Exception {
-                      super.channelActive(ctx);
-                      connectionsCount.countDown();
-                    }
-
-                    @Override
-                    public void channelRead(final ChannelHandlerContext ctx, final Object msg) throws Exception {
-                      final ByteBuf byteBuf = (ByteBuf) msg;
-
-                      while (byteBuf.readableBytes() >= requestMsgSize) {
-                        final String request;
-                        if (partialRequestBytes != null && partialRequestBytes.length > 0) {
-                          final int remainingBytes = requestMsgSize - partialRequestBytes.length;
-                          request = new String(partialRequestBytes) + byteBuf.readCharSequence(remainingBytes, Charset.defaultCharset())
-                                                                             .toString();
-                          partialRequestBytes = null;
-                        } else {
-                          request = byteBuf.readCharSequence(requestMsgSize, Charset.defaultCharset())
-                                           .toString();
-                        }
-
-                        serverReceivedMessagesCount.countDown();
-                        serverReceivedMessage.add(request);
-
-                        final String reply = request + " reply";
-
-                        serverSentMessages.add(reply);
-
-                        final ByteBuf replyBuffer = ctx.alloc()
-                                                       .buffer(reply.getBytes().length);
-
-                        replyBuffer.writeBytes(reply.getBytes());
-
-                        ctx.writeAndFlush(replyBuffer);
-                      }
-
-                      if (byteBuf.readableBytes() > 0) {
-                        partialRequestBytes = new byte[byteBuf.readableBytes()];
-                        byteBuf.readBytes(partialRequestBytes);
-                      }
-                    }
-                  });
-              }
-            })
-            .bind(testPort)
-            .sync()
-            .await();
-  }
-
-}
\ No newline at end of file
+//// Copyright © 2012-2020 VLINGO LABS. All rights reserved.
```

Comment:
@rosetau Why is this commented out?

Reply:
It was just for my earlier commit, it's not commented out in the final commit. Sorry, I shouldn't had to pushed that commit in the first place.